rpc/rpcserver/server.go

```diff
@@ -1161,6 +1169,133 @@ func StartWalletLoaderService(server *grpc.Server, loader *loader.Loader,
 	pb.RegisterWalletLoaderServiceServer(server, service)
 }
 
+// StartTicketBuyerService creates an implementation of the TicketBuyerService
+// and registers it with the gRPC server.
+func StartTicketBuyerService(server *grpc.Server, loader *loader.Loader,
+	tbCfg *ticketbuyer.Config) {
+
+	service := &ticketbuyerLoaderServer{loader: loader, ticketbuyerCfg: tbCfg}
+	pb.RegisterTicketBuyerServiceServer(server, service)
+}
+
+// StartTicketPurchase enables ticket purchases in the ticket buyer.
+func (t *ticketbuyerLoaderServer) StartTicketPurchase(ctx context.Context, req *pb.StartTicketPurchaseRequest) (
+	*pb.StartTicketPurchaseResponse, error) {
+	defer zero.Bytes(req.Passphrase)
+
+	wallet, ok := t.loader.LoadedWallet()
+	if !ok {
+		return nil, grpc.Errorf(codes.FailedPrecondition, "Wallet has not been loaded")
+	}
+	if req.Passphrase != nil {
```

Comment:
I don't see how this could be an optional parameter.  That's just setting us up for failure